packages/next/src/build/swc/options.ts

```diff
@@ -341,14 +341,16 @@ export function getLoaderSWCOptions({
   swcPlugins: ExperimentalConfig['swcPlugins']
   compilerOptions: NextConfig['compiler']
   jsConfig: any
-  supportedBrowsers: string[]
+  supportedBrowsers: string[] | undefined
   swcCacheDir: string
   relativeFilePathFromRoot: string
   bundleTarget: BundleType
   hasServerComponents?: boolean
   isServerLayer: boolean
   isServerActionsEnabled?: boolean
-  optimizeBarrelExports?: string[]
+  optimizeBarrelExports?: {
```

Comment:
This type was previously wrong but it wasn't caught because the loader options weren't used.